entries/ad-layers-dfp/index.js

```diff
@@ -100,7 +100,7 @@ import './style.scss';
         args.targeting = dfpAdDetails[args.format].targeting;
       }
       if (!args.sizeMapping) {
-        if (dfpBuiltMappings && dfpBuiltMappings[args.format]) {
+        if (dfpBuiltMappings) {
           args.sizeMapping = dfpBuiltMappings[args.format];
```

Comment:
do we need a failover here since you aren't checking if the array value is set? Or will it always be set?

Reply:
reverted this change.